src/components/Blocks/Listing/CardWithSlideUpText/CardWithSlideUpTextTemplate.jsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Blocco card con testo animato
+ */
+
+import PropTypes from 'prop-types';
+import { CardReadMore } from 'design-react-kit';
+import cx from 'classnames';
+
+import { UniversalLink } from '@plone/volto/components';
+import { injectLazyLibs } from '@plone/volto/helpers/Loadable/Loadable';
+import {
+  getCalendarDate,
+  getComponentWithFallback,
+} from 'io-sanita-theme/helpers';
+import {
+  ListingCategory,
+  getListingImageBackground,
+  ListingContainer,
+} from 'io-sanita-theme/components/Blocks';
+import { LinkMore } from 'io-sanita-theme/components';
+import { getCategory } from 'io-sanita-theme/components/Blocks/Listing/Commons/utils';
+import { defineMessages, useIntl } from 'react-intl';
+
+import './cardWithSlideUpText.scss';
+
+const messages = defineMessages({
+  vedi: {
+    id: 'card_vedi',
+    defaultMessage: 'Vedi',
+  },
+});
+
+const CardWithSlideUpTextTemplate = (props) => {
+  const intl = useIntl();
+
+  const {
+    items,
+    isEditMode,
+    linkAlign,
+    linkTitle,
+    linkHref,
+    show_type = true,
+    show_section,
+    show_description = true,
+    hide_dates = false,
+    id_lighthouse,
+    linkmore_id_lighthouse,
+    rrule,
+  } = props;
+
+  return (
+    <div className="card-slide-text-template">
+      <ListingContainer data={props} isEditMode={isEditMode}>
+        <div className="grid mb-3 mt-5">
+          {items.map((item, index) => {
+            const image = getListingImageBackground(item, 'large');
+            const category = getCategory(item, show_type, show_section, props);
+            const date = hide_dates
+              ? null
+              : getCalendarDate(item, rrule.rrulestr);
+            const title = item?.title || '';
+
+            const BlockExtraTags = getComponentWithFallback({
+              name: 'BlockExtraTags',
+              dependencies: ['CardWithSlideUpTextTemplate', item['@type']],
+            }).component;
+
+            return (
+              <UniversalLink
+                item={!isEditMode ? item : null}
+                href={isEditMode ? '#' : null}
+                style={
+                  image && {
+                    backgroundImage: `url(${image})`,
+                  }
+                }
+                className="listing-item box bg-img"
+                key={index}
+                data-element={id_lighthouse}
+              >
+                <div className="bg-gradient"></div>
+                {(category || date) && (
+                  <div className="category">
+                    <ListingCategory category={category} item={item} />
```

Comment:
non serve. Su io-sanità ho tolto questo componente perchè qui non esiste la gestione delle categorie fatta come su io-comune. 
Se vogliamo mettere la categoria, devi usare il componente 
CardCategoryBottom come ho usato nelle altre card

Reply:
Ok, ho gestito con il nuovo componente, fatto un paio di CSS per non ripetere le classi e tornare la visualizzazione "in linea" com'era